tasks/header.js

```diff
@@ -0,0 +1,21 @@
+var prependFile = require('prepend-file');
+
+var constants = require('./util/constants');
+
+
+// add headers to dist files
```

Comment:
@bpostlethwaite This `header` task could also update the header of all sources files too.